py/desisim/lya_spectra.py

```diff
@@ -6,9 +6,17 @@
 """
 
 from __future__ import division, print_function
+import pdb
+
+import numpy as np
+from scipy.special import wofz
+
+from pkg_resources import resource_filename
+
+c_cgs = 29979245800.0  # cm/s
```

Comment:
preferable to use `scipy.constants.speed_of_light` (check units) or `astropy.constants.c.to('cm/s').value`